core/template.php

```diff
@@ -244,7 +249,11 @@ function tpl_list($list)
 {
     global $smarty, $config;
     global $listcolumns;
-
+    $smarty->assign('totalresults', 0);
```

Comment:
Move inside the next block as minor optimization if you want

Reply:
✔️